charts/pxc-db/production-values.yaml

```diff
@@ -15,7 +15,7 @@ fullnameOverride: "production"
 
 operatorImageRepository: percona/percona-xtradb-cluster-operator
 
-crVersion: 1.10.0
+crVersion: 1.10.1
```

Comment:
Please revert this to 1.10.0 and just bump the chart version to 1.10.1 since the operator version is staying the same.